drivers/scheduler/k8s/k8s.go

```diff
@@ -1362,6 +1363,18 @@ func (k *K8s) WaitForRunning(ctx *scheduler.Context, timeout, retryInterval time
 //Destroy destroy
 func (k *K8s) Destroy(ctx *scheduler.Context, opts map[string]bool) error {
 	var podList []v1.Pod
+	k8sOps := k8sops.Instance()
+	apRule := ctx.ScheduleOptions.AutopilotRule
+	if apRule.Name != "" {
+		if err := k8sOps.DeleteAutopilotRule(apRule.ObjectMeta.Name); err != nil {
+			if !errors.IsNotFound(err) {
+				return &scheduler.ErrFailedToDestroyAutopilotRule{
+					Name:  apRule.Name,
+					Cause: fmt.Sprintf("%v", err),
```

Comment:
if not using any message, let's use `err.Error()` which already returns a string